### PR TITLE
Spell importer

### PR DESCRIPTION
**Pull request by chrpow**
Adds an importer to import spells from text.

### CODE REVIEW

**Comment by PrototypeESBU** [comment]
Description on your macros needs to be updated. Says "This macro can be used to open the monster importer"
Also you seems to have included changes to a random pack file. Please remove.

File: data/macros/open-spell-importer.js
Comment (Muttley) [inline data/macros/open-spell-importer.js:1]: comment is inaccurate  ;)

File: data/packs/macros.db/open-spell-importer-15E8F4iZ24jq5uE9.json
Comment (Muttley) [inline data/packs/macros.db/open-spell-importer-15E8F4iZ24jq5uE9.json:1]: Again, comment is inaccurate  ;)

File: data/macros/open-spell-importer.js
Comment (chrpow) [reply data/macros/open-spell-importer.js:1]: Fixed.

File: data/packs/macros.db/open-spell-importer-15E8F4iZ24jq5uE9.json
Comment (chrpow) [reply data/packs/macros.db/open-spell-importer-15E8F4iZ24jq5uE9.json:1]: Fixed.

**Review by chrpow — COMMENTED** [review]
All fixed.

**Comment by chrpow** [comment]
Completes #680

**Pull request merged**